iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/NetworkSerDe.java

```diff
@@ -350,6 +350,7 @@ private static void writeBaseNetwork(Network n, NetworkSerializerContext context
         writeLines(n, context);
         writeTieLines(n, context);
         writeHvdcLines(n, context);
+        writeAreas(n, context);
```

Comment:
You need to indicate that `Areas` should only be exported for 1.13+ versions of IIDM:
```suggestion
        IidmSerDeUtil.runFromMinimumVersion(IidmVersion.V_1_13, context, () -> writeAreas(n, context));
```
Or else it won't be possible to export networks containing areas with a previous IIDM version (which could be useful if you have downstream softwares using an older version of PowSyBl).

Reply:
Adressed in 77ae1f322f10dae4af6d17219c9a4586e05ca194